docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - BTC_TXINDEX=1
       - BTC_PRUNE=0
       - BTC_DISABLEWALLET=1
+      - BTC_RPCALLOWIP=172.33.0.0/16
```

Comment:
yeah, I also expose the port in the network settings via the dappnode gui (on the package's port settings page)

I am doing this because I need a connection to the bitcoin rpc port that is reliable for multiple days. I have always had trouble having uninterrupted connections to both the dappnode vpn as well as the dappnode wifi. (In the first case, my vpn connection will occasionally fail, and in the second case, the wifi stops being connected to the internet)

So instead of being on the vpn or dappnode wifi, I just stay on my normal wifi (which uses the normal 192.168.0.0/24 subnet) and assign my dappnode a private static ip address via my home router.


Here's my config sparrow config:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/2193797/167030808-2e5737da-f066-4f88-8b89-3bcda6cc80da.png">

(the proxy config is irrelevant here)


I think that if you are going to set this to something other than `0.0.0.0/0` then you should allow the user to override the configuration. In that case you just need to add the environment variable to the docker-compose file

Reply:
This one still breaks compatibility with other packages like LN I think need confirm from @tropicar also if you're on the same LAN this is unnecessary. Just forward the RPC port to the host from the docker container. You can then point it at the forwarded RPC port on the dappnode using the DAppNodes LAN address. You don't need to use a VPN or the dappnode Wi-Fi to connect like that if the port is properly exposed to the host. No need for proxies and changing the code of the package.